<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>680</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="featureTab">
     <attribute name="title">
      <string>Feature Select    </string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1021</width>
        <height>611</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1017</width>
         <height>607</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="recordingsTab">
     <attribute name="title">
      <string>Recordings        </string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>471</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="folderLabel">
         <property name="text">
          <string>Path to Folders:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="folderPath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Absolute Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectFolder">
         <property name="text">
          <string>Select Folders</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="trackTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>210</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblOverlap">
           <property name="text">
            <string>Overlaping (0 %):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="overlapPercentage">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnTest">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>590</y>
        <width>90</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Windowing:</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Window Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="winSize">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>16</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>280</y>
        <width>469</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>File to Export: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="erportedFile">
         <property name="placeholderText">
          <string>A File Name to Extract Features</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnTestGlobal">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>680</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
  <action name="actionJust_for_fun">
   <property name="text">
    <string>just for fun</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
